src/StylesProvider.js

```diff
@@ -8,7 +8,6 @@ import themeLight from './styles/themeLight';
 
 const themeNameToObject = {
   'default': themeLight,
-  'light': themeLight,
```

Comment:
It is intentional. We used to call this theme 'light' and if this line is removed, it might produce `undefined`.